i18n/vscode-language-pack-zh-hans/translations/main.i18n.json

```diff
@@ -818,7 +818,7 @@
 			"ariaSearchNoResult": "为{1}找到了{0}",
 			"ariaSearchNoResultWithLineNum": "{0} 在 {2} 处找到 {1}",
 			"ariaSearchNoResultWithLineNumNoCurrentMatch": "{0} 已找到 {1}",
-			"ctrlEnter.keybindingChanged": "Ctrl+Enter now inserts line break instead of replacing all. You can modify the keybinding for editor.action.replaceAll to override this behavior."
+			"ctrlEnter.keybindingChanged": "Ctrl+Enter 现在由全部替换改为插入换行。你可以修改editor.action.replaceAll 的按键绑定以覆盖此行为。"
```

Comment:
Multiple line support in Find Widget https://github.com/microsoft/vscode/pull/79329